server/reports/expiring-items/2_3_0/src/template.html

```diff
@@ -16,24 +16,18 @@
     </thead>
     <tbody>
       {% for stockLine in data.stockLines.nodes %}
-      {% set now = now() | date(format="%s") | int %}
-
       <tr>
         <td>{{stockLine.item.code}}</td>
         <td>{{stockLine.item.name}}</td>
         <td class="expiry-date">
-          {% if stockLine.expiryDate %}
-            {% set expiryDate = stockLine.expiryDate | date(format="%s") | int %}
-              {% if expiryDate > now %}
-                {% set diff = (expiryDate - now) / 60 / 60 / 24 | int %}
-                {% if diff <= arguments.monthsItemsExpire * 30 %}
-                  <span class="soon">{{ diff }}</span>
-                {% else %}
-                  <span class="all-good">{{ diff }}</span>
-                {% endif %}
-              {% else %}
-               <span class="expired">Expired</span>
-              {% endif %}
+          {% if stockLine.daysUntilExpired %}
+            {% if stockLine.daysUntilExpired <= 0 %}
```

Comment:
I would add another field 'expiryStatus' or something like that and calculate it in the convert_data

Reply:
I am wanting to wait to refactor further until after https://github.com/msupply-foundation/open-msupply/issues/4702

Reason being is that expiryStatus is a function of monthsItemsExpire which is an argument and currently not accessible in wasm function. 

And this PR is already massive!

Thinking of adding further refactor for another issue post adding sorting (and with that passing arguments into wasm).

What do you reckon?